requests.rest

```diff
@@ -0,0 +1,21 @@
+GET http://localhost:3000/user
+
+###
+
+POST http://localhost:3000/user/create
+Content-Type: application/json
+
+{
+    "name": "Frege", "password": "password"
+
+}
+
+###
+
+POST http://localhost:3000/user/login 
+Content-Type: application/json
+
+{
+    "name": "Frege", "password": "password"
+
+}
```

Comment:
Do we need an extension to use this REST client?

Reply:
Yes, it's called REST Client. The file is not necessary for any functionality, it's just easier to make requests this way.

https://marketplace.visualstudio.com/items?itemName=humao.rest-client